int8 cell_num
float32 center_x
float32 center_y
